### DISCUSSION TITLE
Delete permission for a particular user based on email id on a Sharing Link

### DISCUSSION THREAD

**Discussion by GaddeChaitanya** [pnp/pnpcore#920] [Q&A]
I know there is "DeletePermission" method to remove complete sharing link on a folder or a file, but is there a way to remove particular email or user from GrantedToIdentitiesV2 and GrantedToV2 list.

**Comment by jansenbe** [comment]
@GaddeChaitanya : what you want is possible (see https://docs.microsoft.com/en-us/graph/api/permission-grant?view=graph-rest-1.0&tabs=http), but we've not yet wrapped this functionality in PnP Core SDK. What you can do for the time being is directly make the needed Graph calls using our built-in support for issuing Graph calls: https://pnp.github.io/pnpcore/using-the-sdk/basics-customapirequests.html#making-a-custom-microsoft-graph-request
